db/migrate/20220203235909_create_rule_group_rules.rb

```diff
@@ -0,0 +1,8 @@
+class CreateRuleGroupRules < ActiveRecord::Migration[7.0]
+  def change
+    create_table :rule_group_rules, id: :uuid do |t|
+      t.references :rule_group, type: :uuid, foreign_key: true
+      t.references :rule, type: :uuid, foreign_key: true, index: {unique: true}
```

Comment:
A single rule can belong only into a single group? If no, then the index might be unnecessary.

Reply:
Yeah, a rule will only belong to one rule group because it will only have one parent but a rule group can have many children rules so a `rule_group_id` could show up in the table more than once.